### ISSUE TITLE
1.3b2 Uncaught ReferenceError: PIXI is not defined (client)

### ISSUE THREAD

**Issue by janifulani** [meteor/meteor#5886]
With

`meteor create project && cd project`
`meteor update --release=1.3-modules-beta.2`
`npm init .`
`"dependencies": { "phaser": "^2.4.4" }` added to package.json
`npm install`
`import Phaser from 'phaser';` added to project.js inside client code
`meteor`

I get

```
phaser.js:23027Uncaught ReferenceError: PIXI is not defined
```

In Chrome's console.

Phaser is installed in `node_modules/` and it's package.json has `"main": "./build/phaser.js"`. That JS file seems to contain everything, including a custom PIXI build. I won't use PIXI directly, only through Phaser, so it doesn't make sense to import PIXI (explicitly from 'phaser' or as a dependency in project/package.json) into Meteor?

**Comment by trusktr** [comment]
Possibly related: #5819 #5870

**Comment by ciwolsey** [comment]
Think this might also be linked to #5844

**Comment by avital** [comment]
Should be fixed in `1.3-modules-beta.3`.

**Comment by vadimmelnicuk** [comment]
Same error on beta.16
